doc/website/blog/2022-05-03-announcing-terraform-provider.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Announcing the official Terraform provider for Atlas
+authors: Hedwigz
+tags: [terraform]
+---
+
+Today we are glad to announce the release of the official [Atlas Terraform Provider](https://registry.terraform.io/providers/ariga/atlas).
+
+### What is Terraform
+[Terraform](https://terraform.io/) is a popular open-source tool created by [HashiCorp](https://www.hashicorp.com/) used to greatly simplify the task of provisioning and managing resources in the cloud. With Terraform, organizations can describe the desired state of their infrastructure in simple configuration language and let Terraform plan and apply these changes in an automated way. This way, Terraform allows teams to truly deliver infrastructure-as-code (IaC), which completely changed how teams and organizations manage their cloud infrastructure. 
```

Comment:
```suggestion
[Terraform](https://terraform.io/) is a popular open-source tool created by [HashiCorp](https://www.hashicorp.com/) used to greatly simplify the task of provisioning and managing resources in the cloud. With Terraform, organizations can describe the desired state of their infrastructure in a simple configuration language and let Terraform plan and apply these changes in an automated way. This way, Terraform allows teams to truly deliver infrastructure-as-code (IaC), which completely changed how teams and organizations manage their cloud infrastructure. 
```
Also you say in present `Terraform allows` and switch to past tense in the same sentence `changed how teams`.